### PR TITLE
fix: check UTL verifiedness of tokens before showing spoof warning banner

### PR DESCRIPTION
**Pull request by steveluscher**
fix: check UTL verifiedness of tokens before showing spoof warning banner
  
  # Summary
  
  The UTL provides a `verfied` property. Let's not show a token name/icon spoof warning banner unless that property is `false`.
  
  
  # Test Plan
  
  image
  
  http://localhost:3000/address/2VhjJ9WxaGC3EZFwJG9BDUs9KxKCAjQY4vgd1qxgYWVg/transfers
  
  image
  
  http://localhost:3000/address/DezXAZ8z7PnrnRJjz3wXBoRgixCa6xjnB7YaB1pPB263
  
  image
  
  http://localhost:3000/address/FASTAr1in6u53vykPbCuFTUZnGdiLJrXaTEQ7UAArdBm

### CODE REVIEW

File: app/utils/token-info.ts [hunk 1]
```diff
@@ -107,7 +110,12 @@ export async function getFullTokenInfo(
     ]);
 
     if (!sdkTokenInfo) {
-        return legacyCdnTokenInfo;
+        return legacyCdnTokenInfo
+            ? {
+                  ...legacyCdnTokenInfo,
+                  verified: true,
```
Comment (steveluscher) [inline app/utils/token-info.ts:116]: @mcintyre94, I'm presuming that _everything_ that comes from the legacy token list CDN can be presumed to be ‘verified.’ Does that make sense?

File: app/utils/token-info.ts [hunk 2]
```diff
@@ -123,8 +131,9 @@ export async function getFullTokenInfo(
         logoURI: sdkTokenInfo.logoURI ?? undefined,
         name: sdkTokenInfo.name,
         symbol: sdkTokenInfo.symbol,
-        tags
-    }
+        tags,
+        verified: !!(legacyCdnTokenInfo || sdkTokenInfo.verified),
```
Comment (steveluscher) [inline app/utils/token-info.ts:135]: This means ‘if you're on the legacy token list, or the UTL registry said you're verified, you're verified.’ Is that sound, @mcintyre94?

**Review by askibin — APPROVED** [review]

File: app/utils/token-info.ts (hunk 2 above)
Comment (mcintyre94) [reply app/utils/token-info.ts:135]: Sorry, I missed this PR before! I think this is reasonable, but if the UTL registry contains the same token as unverified maybe that should take precedence?

So:
- Not on the legacy token list at all, UTL verified determines it
- On legacy token list and UTL, UTL verified determines it
- On legacy token list and not UTL, verified

Comment (steveluscher) [reply app/utils/token-info.ts:135]: So, if I understand correctly, that basically implies that we should do this?
```suggestion
        verified: sdkTokenInfo.verified ?? false,
```

1. By this line, we know that `sdkTokenInfo` is non-null.
2. The ‘not on UTL but on legacy list’ case is handled above.

Comment (steveluscher) [reply app/utils/token-info.ts:135]: Building a Vercel preview for that now: #289.

**Pull request merged**